memreps/explicit.py

```diff
@@ -33,4 +33,8 @@ def __invert__(self) -> ExplicitConcept:
         def __iter__(self) -> Iterable[Atom]:
             yield from self.elements
 
+        def subset_of(self, other) -> Atom:
```

Comment:
Type should be: `Optional[Atom]`.

Reply:
Also, can you add this to the `Concept` protocol?

Reply:
👍